google-client-loader.html

```diff
@@ -162,19 +162,22 @@
 
       _fireSuccess: function() {
         this.fire(this.successEventName,
-            { 'name': this.name, 'version': this.version });
+            { 'name': this.name, 'version': this.version },
+            { composed: true });
```

Comment:
It is valid in webcomponents v0. I tested this in the `app-route` youtube demo which was used to diagnose this